cterasdk/edge/backup.py

```diff
@@ -65,6 +65,14 @@ def configure(self, passphrase=None):
 
         logging.getLogger().info('Cloud backup configuration completed successfully.')
 
+    def is_configured(self):
+        """
+        Is Backed configured
+
+        :return bool: True if backup is configured, else False
+        """
+        return bool(self._gateway.get('/config/backup'))
```

Comment:
The correct test should be:
`backup_status = self._gateway.get('/proc/backup/backupStatus')
return backup_status.serviceStatus.id == 'Attached'`
The full enum for all backup status id's is called `BackupConfStatusID` in the appliance's schema

Reply:
Fixed (and added the Enum)